.scripts/replace-catalogs.js

```
@@ -18,34 +20,37 @@ const replaceCatalogDependencies = (packageJsonPath, rootCatalog, catalogs) => {
+    (dependencyType) => {
+      if (packageJson[dependencyType]) {
+        Object.keys(packageJson[dependencyType]).forEach((dependency) => {
+          const version = packageJson[dependencyType][dependency]
+          if (version.startsWith('catalog:')) {
+            const catalogName = version.split(':')[1]
 
-          let resolvedVersion =
-            rootCatalog[dep] || (catalogs[catalogName] && catalogs[catalogName][dep])
+            let resolvedVersion =
+              rootCatalog[dependency] ||
+              (catalogs[catalogName] && catalogs[catalogName][dependency])
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)